code/aepuitemplates/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id("aep-library")
+}
+
+val mavenCoreVersion: String by project
+val aepUiTemplatesModuleName: String by project
+val aepUiTemplatesVersion: String by project
+val aepUiTemplatesMavenRepoName: String by project
+val aepUiTemplatesMavenRepoDescription: String by project
+
+aepLibrary {
+    namespace = "com.adobe.marketing.mobile.aepuitemplates"
+
+    moduleName = aepUiTemplatesModuleName
+    moduleVersion = aepUiTemplatesVersion
+    enableSpotless = true
+    enableCheckStyle = true
+    enableDokkaDoc = true
+
+    publishing {
+        mavenRepoName = aepUiTemplatesMavenRepoName
+        mavenRepoDescription = aepUiTemplatesMavenRepoDescription
+        gitRepoName = "aepsdk-messaging-android"
+        addCoreDependency(mavenCoreVersion)
+    }
```

Comment:
Same comment here. Since we're not publishing this artifact, can we leave this part out?